stdlib/public/runtime/KnownMetadata.cpp

```diff
@@ -80,6 +85,19 @@ namespace ctypes {
     /// with this, but the type isn't copyable, so most of the value
     /// operations are meaningless.
     using BB = ValueBuffer;
+
+    // Types that are defined in the _Concurrency library
+
+    // Default actor storage type.
+    struct BD {
+      void *storage[NumWords_DefaultActor];
+    };
```

Comment:
If these C types are expected to have the same layout characteristics as the Swift types, they need to be aligned.

Reply:
Good catch, thanks